### PR TITLE
LPD-22331 Fixed the logic to show the Tier Prices

### PR DESCRIPTION
**Pull request by welitonpe**

### CODE REVIEW

**Comment by welitonpe** [comment]
ci:test:sf

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/types.d.ts [hunk 1]
```diff
@@ -570,6 +571,13 @@ type ProductSpecification = {
 	value: {[key: string]: string};
 };
 
+type TierPrice = {
+	currency: string;
+	price: number;
+	priceFormatted: string;
+	quantity: number;
+};
+
```
Comment (joanagrego) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/types.d.ts:580]: We already have TierPrices type in `/src/components/LicensePriceCard/LicensePriceChildren.tsx`

**Review by joanagrego — CHANGES_REQUESTED** [review]

**Review by joanagrego — APPROVED** [review]
Good job!

**Review by kevenleone — APPROVED** [review]
LGTM 🔥 

Thank you @welitonpe

**Comment by kevenleone** [comment]
ci:forward

**Comment by kevenleone** [comment]
ci:forward:force

**Pull request closed without merging**